nginx/default.toml

```diff
@@ -19,6 +19,12 @@ worker_processes = 4
 # worker_connections: Connections per Worker Process.  Default = 1024
 worker_connections = 1024
 
+[[include.additional_configuration_file]]
+config_file = "mime.types"
+
+[[include.additional_configuration_file]]
+config_file = "win-utf"
```

Comment:
I don't really understand your question.

I have implemented your original request from last November, "Is it possible/would it be better to have the config variable be filename only, and prepend each file in a list with the include directive? This way, the user passes a list of filenames, not a specifically crafted string, and the template renders the right thing."

The work of specifying a list of standard and non-standard configuration files, is simply to add a bunch of lines like : 

     [[include.additional_configuration_file]]
      config_file = "../../other/stuff/filter_nigerian_spam"